src/test/java/org/fundacionjala/coding/roger/BankOcrTest.java

```diff
@@ -0,0 +1,87 @@
+package org.fundacionjala.coding.roger;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+/**
+ * Unit Tests BankOcr.
+ *
+ */
+public class BankOcrTest {
+
+    private BankOcr bankOcr;
+    /**
+     * SETUP.
+     *
+     */
+    @Before
+    public void setUp() {
+        bankOcr = new BankOcr();
+    }
+    /**
+     * Test convert to number.
+     */
+    @Test
+    public void testOcrToNumber1() {
+        String s1 = "    _  _  _  _  _  _     _ ";
+        String s2 = "|_||_|| || ||_   |  |  | _|";
+        String s3 = "  | _||_||_||_|  |  |  | _|";
+
+        assertEquals("490067713", bankOcr.toNumber(s1, s2, s3));
+    }
+    /**
+     * Test convert to number and verify number.
+     */
+    @Test
+    public void testOcrToNumber2() {
+        String s1 = "    _  _     _  _  _  _  _ ";
+        String s2 = "  | _| _||_||_ |_   ||_||_|";
+        String s3 = "  ||_  _|  | _||_|  ||_| _|";
+        assertEquals("123456789", bankOcr.toNumber(s1, s2, s3));
+        assertEquals("123456789", bankOcr.verify(s1, s2, s3));
+    }
+    /**
+     * Test verify checksum number.
+     */
+    @Test
+    public void testCheckSumTrue() {
+        String s1 = "    _  _     _  _  _  _  _ ";
+        String s2 = "  | _| _||_||_ |_   ||_||_|";
+        String s3 = "  ||_  _|  | _||_|  ||_| _|";
+        assertTrue(bankOcr.checksum(bankOcr.toNumber(s1, s2, s3)));
+    }
+    /**
+     * Test verify checksum number.
+     */
+    @Test
+    public void testCheckSumFalse() {
+        String s1 = " _  _     _  _        _  _ ";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal " _  _     _  _        _  _ " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)